portal/src/main/webapp/config_service.jsp

```diff
@@ -59,6 +59,7 @@
             "priority_studies",
             "show.hotspot",
             "show.oncokb",
+            "show.oncokb.cbiocosmic",
```

Comment:
Naming is incorrect. Also, please separate cbioportal and cosmic options:

```suggestion
            "show.cbioportal",
             "show.cosmic",
```

Reply:
fixed